magpie/api/management/user/user_utils.py

```diff
@@ -135,7 +135,7 @@ def _get_group(grp_name):
     user_params = {"with_param": False}
     if user_checked:
         user_params["with_param"] = True
-        if compare_digest(user_checked.user_name, user_name):
+        if compare_digest(user_checked.user_name.lower(), user_name.lower()):
```

Comment:
We must avoid tweaking the values. That can give timing attack information. If lowercase is enforced anyway from the next version, that will not be necessary.

Reply:
see https://github.com/Ouranosinc/Magpie/pull/596#issuecomment-1804221294